backend/scripts/monitor_ai_processing.py

```
@@ -0,0 +1,338 @@
+        else:
+            return "failed"
+
+    def _analyze_key_points_status(self, ai_result: AIResult = None) -> str:
+        """分析关键要点处理状态"""
+        if not ai_result:
+            return "no_ai_result"
+        
+        if ai_result.key_points:
+            # 检查key_points的内容质量
+            if isinstance(ai_result.key_points, dict):
+                if (ai_result.key_points.get("text") or 
```

Ruff (0.11.9)

Trailing whitespace

Remove trailing whitespace

(W291)